rero_ils/modules/locations/jsonschemas/locations/location-v0.0.1.json

```diff
@@ -170,14 +170,14 @@
         },
         "validation": {
           "messages": {
-            "patternMessage": "The email is not valid."
+            "patternMessage": "The e-mail is not valid."
           }
         }
       }
     },
     "restrict_pickup_to": {
       "title": "Restrict pickup to",
-      "description": "Select locations where items linked to the current location could be requested.",
+      "description": "Select authorised pickup locations for items linked to this location.",
```

Comment:
Seems ok to me, but @zannkukai can you confirm?

Reply:
Seems better 👍